src/app/containers/InlineLink/index.new.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import InlineLink from '../../components/InlineLink';
+import Blocks from '../Blocks';
+import fragment from '../Fragment';
+import { inlineLinkModelPropTypes } from '../../models/propTypes/inlineLink';
+
+const InternalInlineLink = InlineLink.withComponent(Link);
+
+const componentsToRender = { fragment };
+
+const InlineLinkContainer = ({ locator, blocks }) => {
+  const InlineLinkComponent = {
+    Component: InlineLink,
+    props: { href: locator },
+  };
+
+  const regex = '^/news/articles/c[a-zA-Z0-9]{10}o$';
```

Comment:
Should this not be dynamic for any service instead of hardcoded to `news`?

Reply:
Yeah we should use the regex that exists in the route, but I didn't want to change functionality of this file.

Reply:
Could you open an issue to handle the service being dynamic and link it here

Reply:
Placeholder issue https://github.com/BBC-News/simorgh/issues/789